### ISSUE TITLE
ない should have a special part of speech to indicate special inflection for そう

### ISSUE THREAD

**Issue by clindsay3** [JMdictProject/JMdictIssues#11]
ない has an irregular inflection when attached to そう:  なさそう (instead of the normal い-adjective なそう)

Other words with irregularities like this have special part of speech tags, so it seems like that would be appropriate here. As far as I understand, the only other word with this particular formation is よさそう, which already has the adj-ix special part of speech. So I think the simplest solution here would be to add a new part of speech specifically for ない, maybe adj-in?

**Comment by jimmubreen** [comment]
In theory a variant POS could be added, such as adj-in. I'm a little wary of having more POS tags as we may be moving to a different way of handling such things. See http://www.edrdg.org/wiki/index.php/JMdict:_Next_Generation#Entry-wide_Inflection_Pattern_Elements

**Comment by clindsay3** [comment]
Makes sense to do this after the transition to entry-wide inflection elements. 👍
